locales/fr_FR/pcsx2_Main.po

```diff
@@ -2124,28 +2122,28 @@ msgstr "Éditeur de base de données"
 
 #: pcsx2/gui/Panels/GameDatabasePanel.cpp:348
 msgid "Name: "
-msgstr ""
+msgstr "Nom"
 
 #: pcsx2/gui/Panels/GameDatabasePanel.cpp:349
 msgid "Region: "
-msgstr ""
+msgstr "Region"
 
 #: pcsx2/gui/Panels/GameDatabasePanel.cpp:350
 msgid "Compatibility: "
-msgstr ""
+msgstr "Compatibilité"
 
 #: pcsx2/gui/Panels/GameDatabasePanel.cpp:351
 msgid "Comments: "
-msgstr ""
+msgstr "Commentaires"
 
 #: pcsx2/gui/Panels/GameDatabasePanel.cpp:352
 msgid "Patches: "
-msgstr ""
+msgstr "Patches"
 
 #: pcsx2/gui/Panels/GameDatabasePanel.cpp:356
 #: pcsx2/gui/Panels/GameFixesPanel.cpp:25
 msgid "Gamefixes"
-msgstr "Patchs"
+msgstr "Correctif personnalisés"
```

Comment:
Les deux mots doivent être accordés :)

> Correctifs personnalisés

Reply:
Oops honte a moi, merci pour la correction :).
"Oops, thanks for the reply :)"
Can you change it #lightningterror?